android/src/main/java/com/geektime/rnonesignalandroid/RNOneSignal.java

```diff
@@ -357,41 +367,44 @@ public void notificationWillShowInForeground(INotificationReceivedEvent notifica
     }
 
     @ReactMethod
-    public void completeNotificationEvent(final String uuid, final boolean shouldDisplay) {
-        INotificationReceivedEvent receivedEvent = notificationReceivedEventCache.get(uuid);
-
-        if (receivedEvent == null) {
-            Log.e("OneSignal", "(java): could not find cached notification received event with id "+uuid);
+    private void displayNotification(String notificationId){
```

Comment:
displayNotification and preventDefault are private functions. Did they work as expected?

Reply:
Confirmed that these work as expected.